Converters/Revit/Speckle.Converters.RevitShared/RevitRootToHostConverter.cs

```diff
@@ -13,24 +13,69 @@ public record DirectShapeDefinitionWrapper(string DefinitionId, List<GeometryObj
 public class RevitRootToHostConverter : IRootToHostConverter
 {
   private readonly IConverterSettingsStore<RevitConversionSettings> _converterSettings;
+  private readonly ITypedConverter<SOG.Region, string> _regionToFilledRegionConverter;
   private readonly ITypedConverter<Base, List<DB.GeometryObject>> _baseToGeometryConverter;
 
   public RevitRootToHostConverter(
     ITypedConverter<Base, List<DB.GeometryObject>> baseToGeometryConverter,
+    ITypedConverter<SOG.Region, string> regionToFilledRegionConverter,
     IConverterSettingsStore<RevitConversionSettings> converterSettings
   )
   {
     _baseToGeometryConverter = baseToGeometryConverter;
+    _regionToFilledRegionConverter = regionToFilledRegionConverter;
     _converterSettings = converterSettings;
   }
 
   public object Convert(Base target)
   {
-    List<GeometryObject> geometryObjects = _baseToGeometryConverter.Convert(target);
+    List<GeometryObject> geometryObjects = new();
+    List<string> regionIds = new();
 
-    if (geometryObjects.Count == 0)
+    switch (target)
     {
-      throw new ConversionException($"No supported conversion for {target.speckle_type} found.");
+      case SOG.Region region:
```

Comment:
The root to host converter is not the right place to place this logic. Just like there's a base to geometry converter, you should instead add a base to annotation converter that has the region converter.
 You may consider chekcing for active view, to determine whether to use the annotation converter vs geometry converter